frontend/src/components/Settings/settings.jsx

```diff
@@ -1,5 +1,6 @@
 import React from "react";
 import styles from "./settings.module.css"; // Import as a module
+import avatarImage from "./avatar.png";
```

Comment:
Sau này consider lấy hình từ database hoặc từ url nào đó thay vì xài static image file nha em.

Reply:
dạ để em đổi tại cái này em phải xài remove background với lại xài tạm để xem nó có work hay ko ạ

Reply:
thank you anh ạ, để lần sau em bỏ hình vô ạ